src/test/groovy/dk/glasius/course/exercises/simple/Exercises03.groovy

```diff
@@ -9,7 +9,7 @@ import spock.lang.Unroll
 class Exercises03 extends Specification {
     def "simple closure"() {
         given: 'Define a closure that takes one argument'
-            def closure = {  }
+            def closure = { it * 3 }
         when: 'called with a number'
             def result = closure.call(3)
```

Comment:
Din closure fejler, hvis jeg  kalder med 4 og `result == 4` , så din closure burde have været `it * it` men du har forstået ideen i closures.

Reply:
d'oh.